tests/integration/benchmark_parsers.py

```diff
@@ -0,0 +1,110 @@
+import datetime
+import time
+
+from asvdb import BenchmarkResult
+
+
+class Benchmark:
+    """
+    Main general benchmark parsing class
+    """
+
+    def __init__(self, target_id, val=1, split=None):
+        self.name = f"{target_id}"
+        self.val = val
+        self.split = split
+
+    def get_dl_thru(self, full_time, num_rows, epochs, throughput):
+        return create_bench_result(
+            f"{self.name}_dataloader",
+            [("time", full_time), ("rows", num_rows), ("epochs", epochs)],
+            throughput,
+            "rows/second",
+        )
+
+    def bres_loss(self, epoch, loss, l_type="train"):
```

Comment:
nitpick:  I don't think 'bres' adds anything to the name - could we just rename to 'loss' ? If you want to make it clear thats its a BenchmarkResult being returned , we could add type annotations here =) 
```suggestion
    def loss(self, epoch, loss, l_type="train") -> BenchmarkResult:
```